x/protorev/types/genesis.go

```diff
@@ -16,14 +16,18 @@ var (
 			StepSize: sdk.NewInt(1_000_000),
 		},
 	}
-	DefaultPoolWeights = PoolWeights{
-		StableWeight:       5, // it takes around 5 ms to simulate and execute a stable swap
-		BalancerWeight:     2, // it takes around 2 ms to simulate and execute a balancer swap
-		ConcentratedWeight: 2, // it takes around 2 ms to simulate and execute a concentrated swap
-
-		// TODO: This is a temporary weight until we can get a more accurate weight for cosmwasm swaps
-		// ref: https://github.com/osmosis-labs/osmosis/issues/5858
-		CosmwasmWeight: 5, // it takes around 5 ms to simulate and execute a cosmwasm swap
+	DefaultPoolTypeInfo = InfoByPoolType{
+		Balancer: BalancerPoolInfo{
+			Weight: 2, // it takes around 2 ms to simulate and execute a balancer swap
+		},
+		Stable: StablePoolInfo{
+			Weight: 5, // it takes around 5 ms to simulate and execute a stable swap
+		},
+		Concentrated: ConcentratedPoolInfo{
+			Weight:          10, // it takes around 10 ms to simulate and execute a concentrated swap
+			MaxTicksCrossed: MaxTicksCrossed,
```

Comment:
Not sure what the default should be. @NotJeremyLiu

Reply:
Synced in person, will be decided after benchmark testing

Reply:
Updated